heron/api/src/java/com/twitter/heron/streamlet/impl/StreamletImpl.java

```diff
@@ -157,7 +157,7 @@ public String getName() {
    * @param prefix The name prefix of this streamlet
    * @param stageNames The collections of created streamlet/stage names
    */
-  protected void setDefaultNameIfNone(String prefix, Set<String> stageNames) {
+  protected void setDefaultNameIfNone(StreamletNamePrefixes prefix, Set<String> stageNames) {
```

Comment:
Could you rename this enum to `StreamletNamePrefix`? I think it makes more sense that way.

Reply:
Firstly, thanks for review.
Sure, what about `StreamletNamePrefixType` by emphasizing type? In general, most of the enums have type suffix (e.g: `JoinType`)

Reply:
In this case I would say no, because you’re not specifying a type per se but rather just providing a string

Reply:
Addressed and ready for re-review ;)